target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-mr7350.dts

```diff
@@ -175,6 +175,32 @@
 
 		partitions {
 			compatible = "qcom,smem-part";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition-0-devinfo {
+				compatible = "fixed-partitions";
+				label = "devinfo";
+				read-only;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partition@0 {
+					label = "env-data";
+					reg = <0x0 0x40000>;
```

Comment:
This additional partition may be unnecessary.

Reply:
```
kern.err kernel: [    1.342805] devinfo: error parsing ofpart partition /soc@0/nand-controller@79b0000/nand@0/partitions/partition-0-devinfo/nvmem-layout (/soc@0/nand-controller@79b0000/nand@0/partitions/partition-0-devinfo)
```
or do u mean there's other ways of defining this

Reply:
Well, it looks like the kernel tries to parse a nvmem node as a part of fixed-partitions, you need to remove `compatible = "fixed-partitions";` from `partition-0-devinfo {}` when you try the change I suggested.

Reply:
Ah ha, yes, removing that fixed the issue!
Thx